### PR TITLE
DG-42 Displaying my events by id

### PR DESCRIPTION
**Pull request by Mickss**

### CODE REVIEW

File: src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java [hunk 1]
```diff
@@ -71,4 +71,18 @@ public void registerUserForEvent(@RequestHeader(value = "Authorization") String
...
+
+    @GetMapping("/my-events")
+    public ResponseEntity<List<String>> getMyEventIds(
+            @RequestHeader(value = "Authorization") String authorizationHeader) {
+        log.info("Fetching registered event IDs for user");
+        try {
+            UserDTO userDTO = userService.getLoggedInUser(authorizationHeader);
+            List<String> myEventIds = eventRegistrationService.getMyEvents(userDTO.getUserId());
+            return ResponseEntity.ok(myEventIds);
+        } catch (Exception e) {
```
Comment (tknowak) [inline src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java]: We don't need this catch block all. Any error will be logged and mapped to 500 internal server error anyway.

File: src/main/java/org/micks/DiscGolfApplication/events/EventRegistrationService.java [hunk 2]
```diff
@@ -27,4 +30,29 @@ public void registerUserForEvent(String userId, String eventId) {
...
+
+            try (ResultSet resultSet = statement.executeQuery()) {
+                while (resultSet.next()) {
+                    String eventId = resultSet.getString("event_id");
+                    log.debug("Fetched event ID: {}", eventId);
+                    eventIds.add(eventId);
+                }
+            }
+        } catch (SQLException e) {
+            log.error("Error fetching event IDs for user: {}", userId, e);
```
Comment (tknowak) [inline src/main/java/org/micks/DiscGolfApplication/events/EventRegistrationService.java]: This log is not needed as exception thrown below (RuntimeException) will be logged anyway.

**Review by tknowak — CHANGES_REQUESTED** [review]

File: src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java (hunk 1 above)
Comment (Mickss) [reply src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java]: Fixed

File: src/main/java/org/micks/DiscGolfApplication/events/EventRegistrationService.java (hunk 2 above)
Comment (Mickss) [reply src/main/java/org/micks/DiscGolfApplication/events/EventRegistrationService.java]: Fixed

File: src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java [hunk 3]
```diff
@@ -71,4 +71,13 @@ public void registerUserForEvent(@RequestHeader(value = "Authorization") String
         UserDTO userDTO = userService.getLoggedInUser(authorizationHeader);
         eventRegistrationService.registerUserForEvent(userDTO.getUserId(), eventId);
     }
+
+    @GetMapping("/my-events")
+    public ResponseEntity<List<String>> getMyEventIds(
```
Comment (tknowak) [inline src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java]: ResponseEntity is optional in this case and can be removed.

**Review by tknowak — APPROVED** [review]

**Pull request merged**